babel.config.json

```diff
@@ -3,15 +3,16 @@
     ["@babel/preset-env",
       {
         "targets": {
-          "node": 14
+          "node": 18
         }
       }
     ],
     ["@babel/preset-typescript"]
   ],
   "plugins": [
     ["@babel/plugin-proposal-decorators", { "legacy": true }],
-    ["@babel/plugin-proposal-class-properties"]
+    ["@babel/plugin-proposal-class-properties"],
+    ["@babel/plugin-proposal-private-methods"]
```

Comment:
Are these plugins still needed? I think they are part of `preset-env` now?

Reply:
looks like they've been part of preset-env since node 12 and 14.6:
https://github.com/babel/babel/blob/main/packages/babel-compat-data/data/plugins.json#L83
https://github.com/babel/babel/blob/main/packages/babel-compat-data/data/plugins.json#L109

So agreed, these can go

Reply:
Removed them and bumped preset-env version in package.json (although a rebuild should have picked that up I think)

Reply:
I was wrong about this, sorry. The plugins are still needed if we target the legacy decorators, the docs were just worded confusingly: https://github.com/babel/babel/discussions/15912#discussioncomment-6876308

The plugins are built-in in preset-env, so they can be forced on: https://github.com/babel/website/pull/2818/files#diff-166bc972c545d1982d741693a8c9d1bb3abcb653b7b57b9cecb210d13985fd8fR192-R202